beacon_node/beacon_chain/src/data_availability_checker/error.rs

```diff
@@ -35,13 +32,10 @@ pub enum ErrorCategory {
 impl Error {
     pub fn category(&self) -> ErrorCategory {
         match self {
-            Error::SszTypes(_)
-            | Error::MissingBlobs
-            | Error::MissingCustodyColumns
+            Error::MissingBlobs
+            | Error::MissingCustodyColumns(_)
```

Comment:
The categorisation here seem inconsistent with
https://github.com/sigp/lighthouse/blob/9292f82c1c23be6d9a1b9d36015aabf60342518e/beacon_node/network/src/network_beacon_processor/sync_methods.rs#L72

Reply:
Done